docs/features/annotations.md

```diff
@@ -2,28 +2,52 @@
+When the "Add to shared comment pool" option is checked, the comment will be saved to a shared comment pool.
+When creating new annotations, shared comments matching what you have typed will be available via a dropdown list.
+
+![Shared Comments](/images/shared_comments.png)
+
+!!! attention "Deleting annotations"
+    Note that shared comments are tied to the original annotation. If the original annotation is deleted, the comment will not persist in the shared comment pool.
 
-### Scoring Behavior
+### Global annotations
 
-There are two intended ways for course instructors to use the add annotation features. Deductions from maximum, or additions from zero.
+You can can also add a global annotation to a problem that is not tied to a specific line of code.
 
-**Deductions from maximum**
+![Global Annotation Form](/images/annotations_global.png)
 
-Set a `max_score` either programmatically, or under `Edit Assessment > Problems` for the particular non-autograded question. Then when the grader is viewing the code, add negative score, such as `-5` into the score field, to deduct from the maximum. This use case is preferred when grading based on a rubric, and the score is deducted for each mistake.
+To do so, click on the "+" header button corresponding to the problem.
+
+## Scoring Behavior
+
+There are two intended ways for course instructors to use the add annotation features. Deductions from maximum ("negative grading"), or additions from zero ("positive grading").
+
+The default setting is negative grading, but positive grading can be enabled under `Edit Assessment > Problems`.
+
+![Positive Grading](/images/positive_grading.png)
+
+### Negative Grading
+
+Set a `max_score` either programmatically, or under `Edit Assessment > Problems` for the particular non-autograded question. Then when the grader is viewing the code, add a negative score, such as `-5` into the score field, to deduct from the maximum. This use case is preferred when grading based on a rubric, and the score is deducted for each mistake.
 
 The maximum score can be `0` if the deductions are meant to be penalties, such as for poor code style or violation of library interfaces.
 
-**Additions from zero**
+### Positive Grading
+
+Set a `max_score` either programmatically, or under `Edit Assessment > Problems` for the particular non-autograded question. Then when the grader is viewing the code, add a positive score, such as `5` to the score field, to add to the score. This use case is preferred when giving out bonus points.
 
-Set a `max_score` either programmatically, or under `Edit Assessment > Problems` for the particular non-autograded question to `0`. When the grader is viewing the code, add positive scores, such as `5` to the score field, to add to the score. This use case is preferred when giving out bonus points.
+The most common use case is when `max_score` is set to `0`, with marks awarded for each rubric item achieved.
```

Comment:
This line is incorrect. I think with the new positive grading feature, they should just set it to whatever their max grade should be.

Perhaps you could just remove this line?